src/deavinaSJun20/CheckStringIsPalindrome.java

```diff
@@ -0,0 +1,26 @@
+package deavinaSJun20;
+
+import java.util.Scanner;
+
+public class CheckStringIsPalindrome {
+
+	public static void main(String[] args) {
+		Scanner scanner = new Scanner(System.in);
+		System.out.println("Please Enter the String : ");
+		String input = scanner.next();
+		new CheckStringIsPalindrome().checkStringIsPalindrome(input);
+		scanner.close();
+	}
+
+	void checkStringIsPalindrome(String name) {
+		String reverse = "";
+		String name1 = name;
+		for (int i = name.length() - 1; i >= 0; i--) {
```

Comment:
Please practice index as variable instead of 'i';
Rest code is good.

Reply:
made changes as per review comments and updated file checked in